### PR TITLE
Split of Firewall and Erlang Configuration

### PR DESCRIPTION
**Pull request by ufobat**
While installing a CouchDB cluster I was struggeling over some points. With this PR I am hoping to emphasize what configurational steps are actually required required but where hidden in either the Firewall or Configuration Wizard section.

1. Within the part that describes the firewall requirements there was also quite a lot of information that describes *things* regarding Erlang. Therefore I created a subsection regardins all the Erlang *things*

2. Some parts are about testing the communication in/with Erlang, other parts are about actually configurating settings in`vm.args`. In the chapter about using the cluster wizard there was information about the ip|fqdn settings. I think that those settings are always required to be changed for all cluster installation.

3. The configuration curl calls for passwod and bind_address is using `couchdb@<this-nodes-ip-address>`. Previously IP or FQDN was allows in the vm.args file. I think that those 2 parts belong together so I made them consistent.

## Checklist

- [x] Documentation is written and is accurate;
- [ ] `make check` passes with no errors
- [ ] Update [rebar.config.script](https://github.com/apache/couchdb/blob/master/rebar.config.script) with the commit hash once this PR is rebased and merged

### CODE REVIEW

File: src/cluster/setup.rst [hunk 1]
```diff
@@ -45,6 +45,28 @@ possible to force an Erlang application to use a specific port rage.
 This documentation will use the range TCP ``9100-9200``. Open up those ports in
 your firewalls and it is time to test it.
 
+Configure And Test The Communication With Erlang
```
Comment (flimzy) [inline src/cluster/setup.rst]: I suggest using normal "Title Case" capitalization here, to be consistent with the rest of the documentation.

> Configure and Test the Communication with Erlang

File: src/cluster/setup.rst [hunk 2]
```diff
@@ -45,6 +45,28 @@ possible to force an Erlang application to use a specific port rage.
...
+================================================
+
+Make CouchDB use correct IP|FQDN and the open ports.
+----------------------------------------------------
+
+In file etc/vm.args change the the line ``-name couchdb@127.0.0.1`` to
```
Comment (flimzy) [inline src/cluster/setup.rst]: For consistency, "etc/vm.args" should be back-quoted:

> In file ``etc/vm.args`` change...

Also, you duplicated the word "the".

File: src/cluster/setup.rst [hunk 3]
```diff
@@ -45,6 +45,28 @@ possible to force an Erlang application to use a specific port rage.
...
+``-name couchdb@<this-nodes-ip-address|FQDN>`` for each node which defines
+the node and must be seperate for each node. For clustered setup, each node in
```
Comment (flimzy) [inline src/cluster/setup.rst]: This sentence confuses me.  Do you mean "for each line which defines the node"?

And what does "must be separate for each node" mean?   A unique line per node?

**Comment by ufobat** [comment]
any feedback regarding this change

File: src/cluster/setup.rst [hunk 4]
```diff
@@ -45,6 +45,28 @@ possible to force an Erlang application to use a specific port rage.
 This documentation will use the range TCP ``9100-9200``. Open up those ports in
 your firewalls and it is time to test it.
 
+Configure and Test the Communication with Erlang
+================================================
+
+Make CouchDB use correct IP|FQDN and the open ports.
```
[the remark this answers is not in the document]
Comment (flimzy) [reply src/cluster/setup.rst]: Is an FQDN actually required? I would expect that the only requirement is a _resolvable hostname_, which should typically allow for relative hostnames in conjunction with the OS's search path, etc.

If my understanding is correct, can we replace FQDN with simply "host"?

[the remark this answers is not in the document]
Comment (wohali) [reply src/cluster/setup.rst]: Erlang bypasses `/etc/hosts` and local libresolv files, and interprets FQDNs different from "short names", so yes, it has to be an actual FQDN.

**Comment by ufobat** [comment]
I've added the requested changes.

**Comment by wohali** [comment]
@ufobat Thanks! We're waiting for GitHub to catch up with backlogged webhooks so that Travis can run and pass the checks - but this PR is good to go now.
 
Thank you for your contribution to CouchDB! :+1:

**Pull request merged**